platforms/core-runtime/launcher/src/main/java/org/gradle/launcher/cli/converter/LayoutToPropertiesConverter.java

```diff
@@ -84,7 +84,7 @@ private void configureFromHomeDir(File gradleUserHomeDir, Map<String, String> re
     }
 
     private void configureFromBuildDir(BuildLayoutResult layoutResult, Map<String, String> result) {
-        BuildLayout layout = buildLayoutFactory.getLayoutFor(layoutResult.toLayoutConfiguration());
+        BuildLocations layout = buildLayoutFactory.getLayoutFor(layoutResult.toLayoutConfiguration());
```

Comment:
Chose not to rename local vars/parameters etc because `layout` is a fairly decent name, and to reduce impact.